.gitlab/powershell_script_deploy/powershell_script_deploy.yml

```diff
@@ -0,0 +1,30 @@
+# We could (should?) piggy back on deploy_installer_packages_windows-x64 to also deploy this
+# script to $S3_RELEASE_INSTALLER_ARTIFACTS_URI and have the agent-release-management repository
+# publish it to a production bucket like ddagent-windows-stable.
+# For now we can use the dd-agent-mstesting bucket to store the PowerShell script.
+powershell_script_deploy:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/gitlab_agent_deploy$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  stage: choco_and_install_script_deploy
+  rules:
+    !reference [.manual]
```

Comment:
You might need a rule to exclude it from MQ pipeline generation. You can test it with `inv pipeline.test-mq` (non 100% sure of the task name)